src/components/CommentList/CommentList.tsx

```diff
@@ -1,3 +1,18 @@
-export const CommentList: React.FC = () => (
-  <>Put the list here</>
+import React from 'react';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
+import { Comment } from '../../types/Comment';
+import './CommentList.scss';
+
+type Props = {
+  comments: Comment[];
+};
+
+export const CommentList: React.FC<Props> = ({ comments }) => (
+  <ul className="comments" data-cy="comment-list">
+    {comments.map(comment => (
+      <li key={comment.id} className="comments__text">
```

Comment:
move tag li in the commentInfo component

Reply:
ok